### ISSUE TITLE
Extension missing on local, but not on Github Actions

### ISSUE THREAD

**Issue by Spomky** [phpstan/extension-installer#76]
Hi,

I installer `phpstan/extension-installer` and updated my configuration file that now looks like this:

```neon
includes:
    - vendor/phpstan/phpstan/conf/bleedingEdge.neon
    - phpstan-baseline.neon
```

When running the analyse on Github, it works as before and now error is reported.
But on my local env, it reports lots of defects that are all related to the use of deprecations.
When adding back `phpstan/phpstan-deprecation-rules/rules.neon`, it works fine on local but fails on Github arguing the extension is already installed ([see this](https://github.com/web-auth/webauthn-framework/actions/runs/5702344210/job/15453946937)).

I removed and reinstalled all `var`, `vendors` and `*.lock` files, but leads to the same result.

Is there anything I missed to run on my local env?

**Comment by ondrejmirtes** [comment]
So in GitHub Actions the extension-installer works as expected but on your local environment it does not. Can you elaborate on that? What's different?

**Comment by Spomky** [comment]
> Can you elaborate on that? What's different?

The problem is that I don't see any difference between the local environment and that of Github Actions.

On my environment, the following command fails with multiple errors
```
git clone git@github.com:web-auth/webauthn-framework.git
cd webauthn-framework
make st
```

When adding `    - vendor/phpstan/phpstan-deprecation-rules/rules.neon`, the command `make st` is a success.
My guess is that `phpstan/phpstan-deprecation-rules/rules.neon` is not discovered by the extension installer. It seems to be correctly found on Github Actions, but I don't understand why.

**Comment by Spomky** [comment]
OK I think I have it.
There is a conflict between the installer that was installed globally (and I do not use anymore) and the project configuration.
By removing the global dependencies, it works fine.

**Comment by ondrejmirtes** [comment]
Please zip your broken project from your machine and upload it somewhere so I can take a look.
